packages/mongodb-mql-model/src/main/kotlin/com/mongodb/jbplugin/mql/Node.kt

```diff
@@ -94,6 +94,10 @@ data class Node<S>(
      */
     fun copy(componentModifier: (component: Component) -> Component): Node<S> =
         copy(source = source, components = components.map(componentModifier))
+
+    override fun toString(): String {
+        return "Node(components=$components)"
```

Comment:
I've found that toString was serializing the source, which in IntelliJ is the PSIElement and that required readAccess. It doesn't make sense as we are not using the source for anything here, so I'm removing it from the toString.